client/controllers/incidentReports/suggestedIncidentsModal.coffee

```diff
@@ -267,9 +272,15 @@ Template.suggestedIncidentsModal.onCreated ->
         if suspectedAttributes.length > 0
           incident.status = 'suspected'
         incident.url = [@data.article.url]
+        # The disease field is set to the last disease mentioned,
+        # document classification, or event disease with overides in that order.
         event = UserEvents.findOne(@data.article?.userEventId)
         if event?.disease
           incident.disease = event.disease
+        result.diseases.forEach ({name})->
```

Comment:
That would create the disease field when there is no disease which would cause a problem in the suggested field highlighting. Speed isn't a problem here since all of those arrays have only a few items.